src/components/ButtonList.js

```diff
@@ -0,0 +1,11 @@
+function ButtonList({ buttonData }) {
```

Comment:
버튼과 같은 html element들이 모여있는 컴포넌트를 표현할때는 -Group과 같은 표현으로 써주는 것도 좋은 것 같아요~
```suggestion
function ButtonGroup({ buttonData }) {
```